src/release_metadata.rs

```diff
@@ -90,92 +87,63 @@ impl ReleaseMetadata {
         subdir = %subdir.display(),
         description = tracing::field::Empty,
         readme_path = tracing::field::Empty,
-        revision = tracing::field::Empty,
-        revision_count = tracing::field::Empty,
-        commit_count = tracing::field::Empty,
+        %revision,
+        %commit_count,
         spdx_identifier = tracing::field::Empty,
         visibility = ?visibility,
+        %project_id,
+        %owner_id
     ))]
     pub(crate) async fn build(
         flake_store_path: &Path,
         subdir: &Path,
-        revision_info: RevisionInfo,
+        revision: String,
+        commit_count: i64,
         flake_metadata: serde_json::Value,
         flake_outputs: serde_json::Value,
         upload_name: String,
         mirror: bool,
         visibility: Visibility,
-        github_graphql_data_result: GithubGraphqlDataResult,
-        extra_labels: Vec<String>,
-        spdx_expression: Option<spdx::Expression>,
+        labels: Vec<String>,
+        spdx_identifier: Option<spdx::Expression>,
+        project_id: i64,
+        owner_id: i64,
     ) -> color_eyre::Result<ReleaseMetadata> {
         let span = tracing::Span::current();
 
-        span.record("revision_string", &revision_info.revision);
+        if let Some(spdx_identifier) = &spdx_identifier {
+            span.record("spdx_identifier", tracing::field::display(spdx_identifier));
+        }
 
         assert!(subdir.is_relative());
 
-        let revision_count = match revision_info.local_revision_count {
```

Comment:
This code was effectively dead, we'd not actually use `revision_count` anywhere except on L127 on the `span.record`.